be/src/exec/tablet_sink.cpp

```diff
@@ -137,7 +130,7 @@ void NodeChannel::open() {
         request.set_load_mem_limit(_parent->_load_mem_limit);
     }
     request.set_load_channel_timeout_s(_parent->_load_channel_timeout_s);
-    request.set_is_vectorized(_parent->_is_vectorized);
+    request.set_is_vectorized(true);
```

Comment:
Could you set the is_vectorized deprecated in thrift?

Reply:
OK

Reply:
updated, PTAL @chaoyli @wyb